services/sonar/sonar-base.js

```diff
@@ -14,8 +14,9 @@ const modernSchema = Joi.object({
             Joi.number().min(0),
             Joi.allow('OK', 'ERROR')
           ).required(),
-        }).required()
```

Comment:
I found a few scenarios where the measures/metrics array in the response can actually be empty, details below